pkgs/development/libraries/csfml/default.nix

```diff
@@ -1,7 +1,7 @@
 { stdenv, fetchFromGitHub, cmake, sfml }:
 
 let
-  version = "2.4";
+  version = "2.5";
```

Comment:
There seems to be a new release
```suggestion
  version = "2.5.1";
```

Reply:
There only seems to be a new sfml release, csfml doesn't seem to have a 2.5.1 release

Reply:
oh you are right, I mistook it with SFML. Sorry for the noise